src/modules/consts.js

```diff
@@ -0,0 +1,3 @@
+const BaseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const GameUrl = BaseUrl.concat('games/zCRkYjTWW67HHx57jSh1/scores/');
+export default GameUrl;
```

Comment:
- [ ] Great job separating your constants from the code! 💯 However, please make sure you use the correct naming convention for constants which is all uppercase. You can refer to [this article](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx#pre4) if you're curious and want to learn more! 🤓 🔧

Reply:
I am extremely grateful for pointing this out to me and for providing the link. Thanks.